### ISSUE TITLE
Store uploaded images in database

### ISSUE THREAD

**Issue by PrimalPimmy** [osBins/chatApp#108]
Currently images aren't stored in the database

**Comment by sushovanb02** [comment]
@PrimalPimmy @osBins can I work on this?

**Comment by Dhruv1969Karnwal** [comment]
yes, In chat app there should be a image uploader features should be there so I work on it to add this features in you project.
